apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -436,6 +436,30 @@ static Extension createExtensionDate(CcwCodebookInterface ccwVariable, BigDecima
     return extension;
   }
 
+  /**
+   * Helper function to create the valueDate for the specified {@link Extension}.
+   *
+   * @param ccwVariable the {@link CcwCodebookInterface} being mapped
+   * @param date the value to use for {@link Extension#getValue()} for the resulting {@link
+   *     Extension}
+   * @return the output {@link Extension}, with {@link Extension#getValue()} set to represent the
+   *     specified input values
+   */
+  static Extension createExtensionDate(CcwCodebookInterface ccwVariable, LocalDate date) {
+    Extension extension = null;
+    if (date == null) throw new NullPointerException();
```

Comment:
🍰  Can use `Objects.requireNonNull()` for this, built into Java8.

Reply:
Cool - added that to the code.